packages/fast-components/src/styles/display.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from "@microsoft/fast-element";
+
+export type CSSDisplayPropertyValue =
+    | "inline"
+    | "block"
+    | "contents"
+    | "flex"
+    | "grid"
+    | "inline-block"
+    | "inline-flex"
+    | "inline-grid"
+    | "inline-table"
+    | "list-item"
+    | "run-in"
+    | "table"
+    | "table-caption"
+    | "table-column-group"
+    | "table-header-group"
+    | "table-footer-group"
+    | "table-row-group"
+    | "table-cell"
+    | "table-column"
+    | "table-row"
+    | "none"
+    | "initial"
+    | "inherit";
```

Comment:
This could be alphabetized, also I think additional valid values are:
- flow-root
- unset